lib/rules/spaced-comment.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @fileoverview Source code for spaced-comments rule
+ * @author Gyandeep Singh
+ * @copyright 2015 Gyandeep Singh. All rights reserved.
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+
+    // Unless the first option is never, require a space
+    var requireSpace = context.options[0] !== "never";
+
+    // Default to match anything, so all will fail if there are no exceptions
+    var exceptionMatcher = new RegExp(" ");
+
+    // Grab the exceptions array and build a RegExp matcher for it
+    var hasExceptions = context.options.length === 2;
+    var unescapedExceptions = hasExceptions ? context.options[1].exceptions : [];
+    var exceptions;
+
+    if (unescapedExceptions.length) {
+        exceptions = unescapedExceptions.map(function(s) {
+            return s.replace(/([.*+?${}()|\^\[\]\/\\])/g, "\\$1");
+        });
+        exceptionMatcher = new RegExp("(^(" + exceptions.join(")+$)|(^(") + ")+$)");
```

Comment:
My regex is going to allow only the characters in the exceptions list in a given string. If I'm reading your regex correctly, it only allows strings of one exception character repeated one or more times. If that was the intention, then my regex isn't going to work, but examples and unit tests show different exception characters used on the same line.